update: add compatibility guide links for Kotlin 1.7.0 (#3006)
JetBrains/kotlin-web-site: docs/topics/whatsnew17.md

        languageVersion.set(JavaLanguageVersion.of(<MAJOR_JDK_VERSION>)
    }
}
```

## Migrating to Kotlin 1.7.0

### Install Kotlin 1.7.0

> IntelliJ IDEA 2022.1 and Android Studio Chipmunk (212) will suggest updating the Kotlin plugin to 1.7.0 once it is available.

Reviewer: ```suggestion
IntelliJ IDEA 2022.1 and Android Studio Chipmunk (212) automatically suggest updating the Kotlin plugin to 1.7.0.
```
